### PR TITLE
Refresh README for brevity and structure

### PR DESCRIPTION
**Pull request by djfarrelly**
## Description

Improve the README structure and reduce where we can.

[Direct preview URL](https://github.com/inngest/inngest/blob/docs/readme-refresh-july-2022/README.md)

### CODE REVIEW

**Review by tonyhb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,111 +1,87 @@
...
 
 <br />
 
 ## Overview
 
-Inngest makes it simple for you to write delayed or background jobs by triggering functions from events — decoupling your code from your queue.
+Inngest makes it simple for you to write delayed or background jobs by triggering functions from events — decoupling your code from your application.
 
-At a very high level, Inngest does two things:
+At the basic level, this is how you use Inngest:
```
Comment (jpwilliams) [inline README.md:23]: Might be able to remove this sentence entirely and go straight in to the bullets.

File: README.md [hunk 2]
```diff
@@ -1,111 +1,87 @@
...
-We’re open source, committed to preventing vendor-lock in, using simple standards, and build in the open (we like collaboration). Sound interesting?
+- Event-driven systems should be _easy_ to build and adopt
+- Event-driven systems are better than regular, procedural systems and queues
+- Developer experience matters
+- Serverless scheduling enables scalable, reliable systems that are both cheaper and better for compliance
 
-- [**Join our community for support, to give us feedback, or chat with us**](https://www.inngest.com/discord).
-- [Docs](https://www.inngest.com/docs)
-- [Read more about our vision and why this project exists](https://www.inngest.com/blog/open-source-event-driven-queue)
+[Read more about our vision and why this project exists](https://www.inngest.com/blog/open-source-event-driven-queue)
```
Comment (jpwilliams) [inline README.md:37]: This summary and lead-in to the blog post is 🔥

File: README.md [hunk 3]
```diff
@@ -1,111 +1,87 @@
...
 <br />
 
-## Installing Inngest
+## Quick Start
 
-Quick start - this downloads inngest for your os+arch into ./inngest:
+1. Install the Inngest CLI to get started:
 
-```
+```shell
```
Comment (jpwilliams) [inline README.md:45]: Ooh - could add `npm` here?

File: README.md [hunk 4]
```diff
@@ -1,111 +1,87 @@
...
-```json
-{
-  "name": "Post-signup",
-  "id": "abstract-nitty-a14c1d",
-  "triggers": [{ "event": "user/signed.up" }]
-}
+5. Send events to the DevServer. Send right from your application using HTTP + JSON or simply, as a curl with a dummy key of `KEY`.
+
+```shell
+curl -X POST --data '{"name":"test/my.event","data":{"test":true}}' http://127.0.0.1:9999/e/KEY
```
Comment (jpwilliams) [inline README.md:71]: Should we change this to `demo/event.sent` to start moving all of the onboarding events to that namespace?

**Pull request merged**